src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Emitter/CoreBindingHelpers.cs

```diff
@@ -868,22 +868,16 @@ private void EmitBindCoreImplForObject(ObjectSpec type)
                 {
                     case ParsableFromStringSpec stringParsableType:
                         {
-                            if (canSet)
+                            if (canSet && canGet)
```

Comment:
>canGet [](http://example.com/codeflow?start=42&length=6)

I am not getting why need get to be able to emit that?

Reply:
Could you please explain why canGet is needed to allow emitting the binding? examples will help here.